agent/plugin/definition_test.go

```diff
@@ -123,17 +126,18 @@ func TestDefinitionWithoutAdditionalProperties(t *testing.T) {
 		}`),
 	}
 
-	res := validator.Validate(def, map[string]any{
+	cfg := map[string]any{
 		"alpacas": "definitely",
 		"camels":  "never",
-	})
+	}
+	res := validator.Validate(context.Background(), def, cfg)
 
 	if res.Valid() {
-		t.Errorf("validator.Validate(def, {llamas:always,camels:never}).Valid() = true, want false")
+		t.Errorf("validator.Validate(def, %v).Valid() = true, want false", cfg)
```

Comment:
```suggestion
		t.Errorf("validator.Validate(def, % #v).Valid() = true, want false", cfg)
```

makes the difference between:
```
map[alpacas:definitely camels:never]
```
and
```
map[string]string{"alpacas":"definitely", "camels":"never"}
```

the latter of which is much easier to copypaste if necessary, and also much easier to read, IMO